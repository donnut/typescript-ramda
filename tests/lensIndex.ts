import R_lensIndex = require('../ramda/dist/src/lensIndex');

declare const number: number;

// @dts-jest:pass:snap
R_lensIndex(0);
// @dts-jest:pass:snap
R_lensIndex(number);
