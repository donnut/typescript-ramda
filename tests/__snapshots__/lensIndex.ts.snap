// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.lensIndex(0) 1`] = `"PseudoLens<0>"`;

exports[`R.lensIndex(number) 1`] = `"PseudoLens<number>"`;
